packages/web/src/utils/helpers.ts

```diff
@@ -333,3 +334,11 @@ export const scrollToTop = (
     element.scrollIntoView({ behavior: 'smooth' })
   }, 100)
 }
+
+// TEXT HELPERS
+
+export const consolidateAmounts = (str: string | undefined) => {
+  const regex = /(?<=\d)\s(?=\d)|(?<=\d)\s(?=\D)/g
+  const text = str?.replace(regex, '\u00a0')
+  return text
```

Comment:
suggestion: you can return directly `str?.replace(regex, '\u00a0')` without create a new variable (how use memory)

Reply:
thanks: now done [here](https://github.com/betagouv/transition-ecologique-entreprises-widget/pull/189/commits/f605a0e6c1fe44b8f9c57dc8507da5cc20fc380d)